.changeset/big-parrots-sleep.md

```diff
@@ -0,0 +1,9 @@
+---
+'@asgardeo/javascript': patch
+'@asgardeo/browser': patch
+'@asgardeo/nextjs': patch
+'@asgardeo/node': patch
```

Comment:
if we are planning to keep node sdks as well, we may need to change the repo name to reflect that, IMO.

Reply:
The repo will be changed to `asgardeo/javascript`

Hope this'll resolve issues :)